test('pass!', () => {

})

test('This should fail', () => {
    throw new Error('Failure!')
})